dwhdelft.nl/pages/mixup.vue

```diff
@@ -0,0 +1,319 @@
+      eens gewoon een bar zijn, is het heel gezellig bijkletsen en misschien een dansje wagen.
+    - MIXUP wordt volledig mogelijk gemaakt door een team enthousiaste vrijwilligers van DWH. Ben je enthousiast geworden en wil je
+      bijdragen aan en onderdeel worden van een fantastische community? Wordt dan lid.
+    - Bang om alleen te komen? We kunnen je aan een barbuddy koppelen om te leren kennen, die je ook aan anderen kan voorstellen!
+  invite:
+    announcement: 'Kom gerust langs op onze baravond:'
+    time: Elke Zaterdag vanaf {0}
+  membership_button: Schrijf je nu in bij DWH
+  barbuddy_button: Vind een buddy
+  instagram: Onze evenementen en UPdates
+  months:
+    0: januari
+    1: februari
+    2: maart
+    3: april
+    4: mei
+    5: juni
+    6: juli
+    7: augustus
+    8: september
+    9: oktober
+    10: november
+    11: december
+  bulletPoints:
+    - title: Dansavonden
+      description: Elke week komen we samen voor drankjes, vaak is er zelfs een DJ om die voetjes op te bewegen!
+      image: events-dancenights
+    - title: Feesten
+      description: Disco, hard rock, Oktoberfest, Halloween en nog veel meer... Er is altijd tijd voor een feestje.
+      image: events-parties
+    - title: Drag optredens
+      description: Ons eigen drag huis HAUS of 4D geeft elke twee maanden geweldige shows
+      image: events-dragshow
+    - title: Evenementen
+      description: Karaoke, proeverijen, vinyl nachten of een talentenshow, er is veel te beleven op een MIXUP avond.
+      image: events-silentdisco
+  highlights:
+    title: Onze Hoogtepunten
+    events:
+      - title: MIXUP Oorsprong
+        description: Het allereerste feestje onder de MIXUP banner was MIXUP Origin’s, het openingsfeest! Miss Abby OMG gaf ons een prachtig optreden, het feest was aan!
+        image: highlight-abbyomg
+      - title: Zomerfeest
+        description: Heet weer en nog hetere mensen! DJ Maarten verzorgde tropische tunes terwijl we zomercocktails aan het nippen waren en afkoelde in een daadwerkelijk zwembad!
+        image: highlight-summer-party
+</i18n>
+
+<script setup>
+import { warn } from 'vue'
+
+useHead({
+  bodyAttrs: {
+    class: 'mixup-colors',
+  },
+})
+
+const { t, tt } = useT()
+
+const { data: openingHours } = await useAsyncData(() => queryContent('opening_hours').findOne())
+const barOpeningHours = openingHours.value.events.find((event) => event.day.en === 'Saturday')
+
+const { image: imageIcons } = useDynamicImages(
+  import.meta.glob('~/assets/images/photos/mixup/icons/*', { eager: true })
+)
+const { image: imageOverviews } = useDynamicImages(import.meta.glob('~/assets/images/photos/mixup/*', { eager: true }))
+
+const instagramChannelsMixup = [
+  {
+    name: 'MIXUP',
+    widgetId: '219aedf13a4355fe95c6e27ed1f7386c',
+    instagram: 'mixupdelft',
+  },
+]
+
+const { data: events } = await useAsyncData('events', async () => {
```

Comment:
Probably a good idea to extract the whole events thing to its own component since it's quite verbose :)